tests/test_helper/microovn.bash

```diff
@@ -294,3 +314,83 @@ function wait_microovn_online() {
 
     return $rc
 }
+
+# wait_ovsdb_cluster_changes_applied CONTAINER CONTROL_PATH DB_NAME TIMEOUT
+#
+# Wait until OVN OVSDB cluster member converges with rest of the cluster. This function
+# checks output of ovn-appctl to make sure that field 'Entries not yet applied' reaches 0.
+# It requires CONTROL_PATH which points to database's .ctl file (i.e. path/to/ovnnb_db.ctl)
+# and DB_NAME which should be either "OVN_Northbound" or "OVN_Southbound.
+#
+# TIMEOUT in seconds is roughly obeyed. If conditions are not met before timeout is reached, this
+# functions returns non-zero RC
+function wait_ovsdb_cluster_changes_applied() {
```

Comment:
nit: I know that it did not exist when you wrote this, just wanted to highlight that there is now a generic `wait_until` function in `common` that we could consider to migrate to using when we get time for it.